.github/workflows/release.yml

```
@@ -0,0 +1,118 @@
+        with:
+          status: success
+          text: '🚀 New SDK release published to npm!'
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  # Job to create GitHub releases when packages are published
+  create-github-release:
+    name: Create GitHub Release
+    needs: release
+    runs-on: ubuntu-latest
+    if: needs.release.outputs.published == 'true'
```

actionlint (1.7.7)

property "published" is not defined in object type {}

(expression)